blockchain/My-Wallet-V3-iOS
# feat(NetworkManager): IOS-1431 disable public key pinning for staging and dev …

…environments

## Objective

Eliminate extra step in the build process.

## Description

Added `DISABLE_CERT_PINNING` flag to dev and staging envs.

## How to Test

No changes to the `.xcconfig` files are necessary.
- Run in dev and staging, a breakpoint at https://github.com/blockchain/My-Wallet-V3-iOS/compare/kevin/IOS-1431/cert_pinning?expand=1#diff-c8cd3b7d05dc3f804b27e6e1eeb27c22R161 should be hit.
- Run in production, a breakpoint at https://github.com/blockchain/My-Wallet-V3-iOS/compare/kevin/IOS-1431/cert_pinning?expand=1#diff-c8cd3b7d05dc3f804b27e6e1eeb27c22R163 should be hit.

## Screenshot/Design assessment

N/A

## Merge Checklist
